support-frontend/package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "clean": "echo 'Cleaning Public Folder' && rimraf public/compiled-assets",
     "validate": "echo 'Validating JS' && npm-run-all 'lint --fix' flow && yarn orphans",
+    "lint:nibble": "eslint-nibble ./assets --ext .ts,.tsx",
```

Comment:
I've left this is as it was really useful whilst doing this work. It gives a nice interface to running eslint with just one specific rule. It helped me fix up all the errors I was introducing without fixing up all the existing errors too and bloating this PR even more!